### DISCUSSION TITLE
Remove themes that are not part of the package

### DISCUSSION THREAD

**Discussion by DiegoSof2** [prasanthrangan/hyprdots#88] [Ideas]
I noticed that in addition to hyprdots themes, there are other themes in rofi and qt5 settings like iggy, blue by qbal etc, I believe it would be interesting to remove the others and leave only the package themes for the package to be cleaner

**Comment by prasanthrangan** [comment]
Hi, I don't see any other themes in rofi and qt5 settings. Maybe its from other configs you might have copied.

**Reply by DiegoSof2** [reply]
mine lists all these
![swappy-20230819_144501](https://github.com/prasanthrangan/hyprdots/assets/105827698/6f2a2828-b690-41cc-95f8-e4d27606613f)
![swappy-20230819_144521](https://github.com/prasanthrangan/hyprdots/assets/105827698/78ec7c47-6a3b-4b19-ac29-c188db829667)
I did a minimal installation of arch linux then hyprdots, but if other themes do not appear, it means that there is a problem with my installation

**Reply by prasanthrangan** [reply]
> mine lists all these

Oh, dont worry about these. These themes are part of rofi itself.
hyprdots rofi styles and themes are in `~/.config/rofi/` only, and you can select it by `Super` + `shift` + `A`
